src/api/route-services/experiment.js

```diff
@@ -24,24 +24,31 @@ const {
   convertToDynamoUpdateParams,
 } = require('../../utils/dynamoDb');
 
+const experimentServiceLogValue = '[ExperimentService] - ';
+
 class ExperimentService {
   constructor() {
     this.experimentsTableName = `experiments-${config.clusterEnv}`;
     this.cellSetsBucketName = `cell-sets-${config.clusterEnv}`;
     this.processedMatrixBucketName = `processed-matrix-${config.clusterEnv}`;
     this.rawSeuratBucketName = `biomage-source-${config.clusterEnv}`;
+    this.filteredCellsBucketName = `biomage-filtered-cells-${config.clusterEnv}`;
 
     mockData.matrixPath = mockData.matrixPath.replace('BUCKET_NAME', this.rawSeuratBucketName);
     this.mockData = convertToDynamoDbRecord(mockData);
   }
 
   async getExperimentData(experimentId) {
+    logger.log(`${experimentServiceLogValue}GET experiment ${experimentId} data`);
```

Comment:
I added logs for the experiment service because the ones we have are lacking a little bit, if people think this is a good idea I'll try to add similar logs to the other services.

Reply:
Also there might be a better way to add this logs, so please feel free to suggest better options

Reply:
Would it be cleaner if you create a factory function that would return a function which wrapps `logger.log` with an initialized prepended values?

Reply:
Sorry, I mean appended not prepended

Reply:
done!